### PR TITLE
log environmentBlob in the experimentRun

### PR DESCRIPTION
**Pull request by anandjakhaniya**
**Jira Ticket:** https://vertaai.atlassian.net/browse/VR-9357

### CODE REVIEW

**Comment by anandjakhaniya** [comment]
Compilation error because protos are not merge in the master

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java [hunk 1]
```diff
@@ -3050,4 +3051,33 @@ public ExperimentRun cloneExperimentRun(
...
+                .setCode(Code.NOT_FOUND_VALUE)
+                .setMessage(ModelDBMessages.EXP_RUN_NOT_FOUND_ERROR_MSG)
+                .build();
+        throw StatusProto.toStatusRuntimeException(status);
+      }
+      experimentRunEntity.setEnvironment(ModelDBUtils.getStringFromProtoObject(environmentBlob));
+      session.update(experimentRunEntity);
+      transaction.commit();
+      LOGGER.debug("EnvironmentBlob logged successfully");
+    } catch (Exception ex) {
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java:3075]: Where is the permission check being done? I only see direct DB operations.

**Review by conradoverta — CHANGES_REQUESTED** [review]

File: backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java:3075]: Ohh Sorry, i forget to add it.

Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java:3075]: Now added check on the service https://github.com/VertaAI/modeldb/pull/1786/files#diff-73f4a3ee85b589c5a06a8200ac022bf55ffc3f201c9b4abe78bd9734b946da44R2806

**Review by conradoverta — APPROVED** [review]

**Pull request merged**